stylesheets/components/Cards/Cards.css

```diff
@@ -0,0 +1,214 @@
+
+    /* Card CSS */
+    .card {
+        position: relative;
+        min-width: 10.625rem;
+        max-width: 35rem;
+        min-height: 9.375rem;
+        max-height: 400px;
+        background-color: white;
+        border-radius: 12px;
+        box-shadow: var(--box-shadow);
+    }
+
+    .card--small {
+        width: min-content;
+        height: min-content;
+    }
+
+    .card__img--small {
+        width: 10.625rem;
+        height: 10.625rem;
+    }
+
+    .card__img--small img {
+        border-radius: 12px;
+    }
+
+    .card__content--column {
+        display: flex;
+        flex-direction: column;
+        padding: 10px 12px;
+    }
+
+    .card__content--row {
+        display: flex;
+        flex-direction: row;
+        padding: 10px 12px;
+    }
+
+    .card__icon {
+        position: absolute;
+        outline: 0px;
```

Comment:
In case of value to be 0, We can avoid using units.

Reply:
Resolved in the latest commit. Thank you.